src/mlang/mpp_ir/mpp_ir_to_bir.ml

```diff
@@ -109,7 +109,7 @@ let var_filter_compatible_subtypes (subtypes : Mir.variable_subtype list)
           List.exists
             (fun st ->
               match (st : Mir.variable_subtype) with
-              | Base | GivenBack | Computed -> true
```

Comment:
Out of the loop here, could you explain this ?

Reply:
So called "given back" (restituee) variables are not really a variable subtype by themselves, rather an attribute. Furthermore, it does not only apply to "computed" variables : there are almost 1000 "input" (saisie) variables that are also "given back" (because, you know, WTF...). Anyways, removing "GivenBack" here fixes a bug (mlang-generated verification chainings called more individual verifications than compir-generated verifications chanings).